# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../vx32/hotfix.patch
# Copyright (C) 2009 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- a/src/Makefrag.vanilla	2009-12-16 12:39:17.000000000 +0100
+++ a/src/Makefrag	2009-12-16 12:39:24.000000000 +0100
@@ -28,7 +28,7 @@
 VX32_CFLAGS := -nostdinc -Ilibvxc/include $(COMMON_CFLAGS)
 VX32_LDFLAGS := -Llibvxc -L$(dir $(shell vx32-gcc -print-libgcc-file-name)) \
 		$(COMMON_LDFLAGS) libvxc/vx32/crt0.o
-VX32_LDLIBS := -lc -lgcc
+VX32_LDLIBS := -lc `$(CC) -print-libgcc-file-name`
 
 # Set compiler options if we should generate x87 FPU code.
 HAVE_X87 = yes
--- a/rc/vxa/Makefrag.vanilla	2009-12-16 12:41:45.000000000 +0100
+++ a/src/vxa/Makefrag	2009-12-16 12:41:52.000000000 +0100
@@ -4,6 +4,5 @@
 include vxa/bz2/Makefrag
 include vxa/jpeg/Makefrag
 include vxa/jp2/Makefrag
-include vxa/vorbis/Makefrag
 include vxa/flac/Makefrag
 
